### PR TITLE
Html updates

### PR DESCRIPTION
**Pull request by ruthtillman**

### CODE REVIEW

File: app/views/pages/search_tips.html.erb [hunk 1]
```diff
@@ -30,6 +30,7 @@
 <h2>Advanced Search</h2>
 
 <p>You can also use the <%= link_to 'Advanced Search', page_path('advanced') %> to construct a complicated search. For example, if you only want books about homelessness published after 2000, you'd use the following fields:</p>
+
```
Comment (banukutlu) [inline app/views/pages/search_tips.html.erb]:
```suggestion
```

File: config/locales/blacklight.en.yml [hunk 2]
```diff
@@ -41,4 +44,4 @@ en:
     array:
       words_connector: "<br>"
       two_words_connector: "<br>"
-      last_word_connector: "<br>"
\ No newline at end of file
+      last_word_connector: "<br>"
```
Comment (banukutlu) [inline config/locales/blacklight.en.yml]:
```suggestion
      last_word_connector: "<br>"
  
```

File: config/locales/blacklight.en.yml [hunk 3]
```diff
@@ -41,4 +44,5 @@ en:
...
```
Comment (banukutlu) [inline config/locales/blacklight.en.yml]:
```suggestion
      last_word_connector: "<br>"
      
```

File: config/locales/blacklight.en.yml [hunk 4]
```diff
@@ -41,4 +44,5 @@ en:
...
+
```
Comment (banukutlu) [inline config/locales/blacklight.en.yml]:
```suggestion
```

File: config/locales/blacklight.en.yml [hunk 5]
```diff
@@ -16,6 +16,9 @@ en:
         title: 'Refine search results'
       continue_search:
         title: 'Or continue your search in'
+      zero_results:
+        remove_fields: ''
```
Comment (banukutlu) [inline config/locales/blacklight.en.yml]: @ruthtillman is `remove_fields` supposed to be empty, if so why are we adding it?

File: config/locales/blacklight.en.yml [hunk 6]
```diff
@@ -16,6 +16,9 @@ en:
...
+        use_fewer_keywords: 'Use X to remove sUse fewer keywords to start, then refine your search using the links on the left.'
```
[the remark this answers is not in the document]
Comment (ruthtillman) [reply config/locales/blacklight.en.yml]: Committed the wrong version 💀 I'll fix when I'm in shortly

[the remark this answers is not in the document]
Comment (ruthtillman) [reply config/locales/blacklight.en.yml]: Same with remove_fields being blank. 🙃

File: config/locales/blacklight.en.yml (hunk 5 above)
Comment (ruthtillman) [reply config/locales/blacklight.en.yml]: Sorry, y'all, I'd saved but AFTER the commit for some reason. Pulled, committed, updated, pushed... hoping the newest version is now ok.

**Pull request merged**